rhobs/alerting/data_plane/prometheus.serviceprovider_alerts.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-spi-alerting
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+    - name: serviceprovider_alerts
+      interval: 1m
+      rules:
+        - alert: ServiceProvider5xxErrorsRate
+          expr: |
+            (
+              sum without (status) (increase(redhat_appstudio_spi_service_provider_response_time_seconds_count{status=~"5.."}[1h]))
```

Comment:
Need a test case in which we have more than one 5xx time series to see that the summing works

Reply:
Added